.circleci/config.yml

```diff
@@ -2,6 +2,28 @@ version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
 
+jobs:
+  lint_frontend:
+    working_directory: ~/membrane_live/assets
+    docker:
+      - image: cimg/node:18.4.0
+
+    steps:
+      - checkout:
+          path: ~/membrane_live
+      - run:
```

Comment:
Oh, one more thing (although can be done outside of this PR) - caching. The job takes 12s now, so it's not bad, but I guess it will grow and persisting `node_modules` or at least `~/.npm` might be useful

Reply:
OK, I'll add an issue to Jira

Reply:
https://membraneframework.atlassian.net/browse/MV-267